bgc_data/nrs_depth_binned_ctd_data.sql

```diff
@@ -0,0 +1,93 @@
+-- Materialized view for nrs depth binned ctd product
+CREATE MATERIALIZED VIEW nrs_depth_binned_ctd_data AS
+WITH
+--modify the site code to match with the deployments table
+--consider only NRS stations
+  nrs_trips AS (
+      SELECT
+         z.stationname AS "StationName",
+         z.trip_code AS "TripCode",
+         z.latitude AS "Latitude", 
+         z.longitude AS "Longitude",
```

Comment:
Also there's no need to bring in latitude & longitude here and then have to carry them all the way through. All you need to match up NRS trips with CTD profiles is `projectname`, `stationcode`, and `sampledatelocal`. 
Any other trip metadata (or cast metadata) can just be added in at the very end.

Reply:
updated in the next commit